lib/browser/_asserters.js

```diff
@@ -131,13 +131,41 @@ exports.fuzzyString = function assertFuzzyString(
   });
 };
 
-exports.elementsNumber = function assertElementsNumber(num, selector) {
+/**
+ *
+ * @param {number|{?equal: number| ?min: number| ?max: number}} numberOrObj
+ * @param {string} selector
+ * @return {Asserter}
+ */
+exports.elementsNumber = function assertElementsNumber(numberOrObj, selector) {
   return new Asserter(elems => {
+    let error;
     const found = elems.length;
-    if (found !== num) {
-      throw new AssertionError(
-        `${selector} should match ${num} elements - actually found ${found}`
-      );
+    if (typeof numberOrObj === 'number') {
+      if (found !== numberOrObj) {
+        error = new AssertionError(
+          `selector "${selector}" should match ${numberOrObj} elements - actually found ${found}`
+        );
+      }
```

Comment:
how about:
```js
if (typeof numberOrObj === 'number') numberOrObj = { equal: numberOrObj };
```

Reply:
any particular reason for assigning `error` just to `throw` later?

Reply:
no particular reason - likely it was some left over of a first idea how to implement the logic